e2e/monitor/main_test.go

```diff
@@ -44,7 +44,7 @@ import (
 const (
 	localnetPrivateKey = "dfe61681b31b12b04f239bc0692965c61ffc79244ed9736ffa1a72d00a23a530"
 	retries            = 10
-	btcAddress         = "mw47rj9rG25J67G6W8bbjRayRQjWN5ZSEG"
+	btcAddress         = "mv5gj33YaFviPFDmkkUpb31C4uxoB4ZZ5D"
```

Comment:
Not for this PR, but @ClaytonNorthey92 is there a reason you have this address in a `.env` file above and hardcoded here?  Might be better to only specify once.

Reply:
@jcvernaleo likely no good reason.  we can just use the `.env` file